### PR TITLE
Add Producthunt extension

### PR DESCRIPTION
**Pull request by ragnorc**
## Description

New Extension: Producthunt

## Type of change

- New extension

### CODE REVIEW

File: extensions/producthunt/package.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "producthunt",
+  "title": "Product Hunt",
+  "description": "Read the latest stories of Hacker News.",
```
Comment (loris) [inline extensions/producthunt/package.json:4]: You probably meant Product Hunt here

Comment (ragnorc) [reply extensions/producthunt/package.json]: Yep, fixed!

File: extensions/producthunt/package.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  "name": "producthunt",
+  "title": "Product Hunt",
+  "description": "Get the latest products from the fontpage of Product Hunt.",
+  "icon": "icon.png",
+  "author": "Ragnor Comerford",
+  "commands": [
+    {
+      "name": "frontpage",
+      "title": "Check Today's Product Hunt",
+      "subtitle": "Product Hunt",
```
Comment (thomaspaulmann) [inline extensions/producthunt/package.json:11]: We tend to minimize repeatetive names here. Both are shown in the root search. How about naming it like this?
```suggestion
      "title": "Check Today's Hunts",
      "subtitle": "Product Hunt",
```

**Review by thomaspaulmann — CHANGES_REQUESTED** [review]
Nice addition. For next time, please attach a screenshot. Otherwise it's hard to reason about the extension. In this case it was easy because the code looks familiar ;-) 

I left a few minor comments to name the extension better to fit our UI guidelines.

**Pull request merged**